lib/src/widgets/common/primary_scaffold.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import '../../config/colors.dart';
+
+class BaseScaffold extends Scaffold {
+  BaseScaffold({
+    super.key,
+    super.appBar,
+    super.floatingActionButton,
+    super.floatingActionButtonLocation,
+    super.floatingActionButtonAnimator,
+    super.persistentFooterButtons,
+    super.persistentFooterAlignment = AlignmentDirectional.centerEnd,
+    super.drawer,
+    super.onDrawerChanged,
+    super.endDrawer,
+    super.onEndDrawerChanged,
+    super.bottomNavigationBar,
+    super.bottomSheet,
+    super.backgroundColor,
+    super.resizeToAvoidBottomInset,
+    super.primary = true,
+    super.drawerDragStartBehavior = DragStartBehavior.start,
+    super.extendBody = false,
+    super.extendBodyBehindAppBar = false,
+    super.drawerScrimColor,
+    super.drawerEdgeDragWidth,
+    super.drawerEnableOpenDragGesture = true,
+    super.endDrawerEnableOpenDragGesture = true,
+    super.restorationId,
+    required this.body,
+  }) : super(
+          body: Container(
+            decoration: BoxDecoration(
+              gradient: LinearGradient(
+                begin: Alignment.bottomCenter,
+                end: Alignment.topCenter,
+                colors: [
+                  Colors.white,
+                  AppColors.kBlueColor.shade25,
+                ],
+              ),
+            ),
+            child: body,
+          ),
+        );
+  final Widget body;
```

Comment:
make gradient a property of this Scaffold which we can set from wherever this is used, we can also set the current values as default values.

Reply:
pupose :-  we can use in overallApp in future , make use more open for scaffold to give same functionality through out the app , sb apne control mae hoga